evm/vm/forks/sharding/contracts/validator_manager.v.py

```diff
@@ -28,11 +28,10 @@
 # Number of validators
 num_validators: public(int128)
 
-# Collation headers
-collation_headers: public({
-    parent_hash: bytes32,
-    score: int128,
-}[bytes32][int128])
+# Collation headers: (parent_hash || score)
+# parent_hash: 24 bytes
+# score: 8 bytes
```

Comment:
Last time (March 5th) I heard from V it's 26 bytes + 6 bytes for JMRS, so did it changed to 24 bytes + 8 bytes?

Reply:
I remember the 24+8 I heard was before March 5th so I guess 26+6 is the latest one.